### ISSUE TITLE
iOS ARM64 core does not point at the right libraries

### ISSUE THREAD

**Issue by jet082** [libretro/flycast#621]
So, currently, trying to build the iOS core for arm64 does not work at all. One problem it has is that it pulls in the wrong OpenGL libraries.

```
In file included from ./core/libretro-common/include/glsm/glsm.h:30:
./core/libretro-common/include/glsym/rglgen_headers.h:44:10: fatal error: 'OpenGL/gl3.h' file not found
#include <OpenGL/gl3.h>
```

It should be pulling in OpenGLES libraries. Fixing this is not enough to make it build for iOS, but it's a step.

**Comment by flyinghead** [comment]
Upstream doesn't support iOS and the arm64 dynarec has never been used on this platform. So it's going to be a long and bumpy road to get iOS support.

**Comment by inactive123** [comment]
I believe it did support iOS at some point but this is like in iOS6 days
and Apple breaks ARM ABI all the time, so no point in catering to it.

However, we could try the generic recompiler on iOS. The more powerful iOS
devices might be able to run games acceptably with that.

**Comment by jet082** [comment]
There is a lot of iOS stuff in the Makefile. Honestly, I've gotten it fairly close to compiling myself just by editing that and a few files.

However, things got a bit too sloppy (since I don't know how to set certain flags in the Makefile, I ended up just editing various header files to force the settings I needed) and I abandoned it.

My impression, having fixed 10+ cores for ARM64 now, is that it probably isn't too bad... But maybe a bit beyond my particular skill level.
